--- origsrc/xmlrpc-epi-0.54/configure.in	2008-04-16 05:38:01.000000000 -0500
+++ src/xmlrpc-epi-0.54/configure.in	2008-05-07 16:29:23.734375000 -0500
@@ -3,8 +3,6 @@
 AC_INIT(src/xmlrpc.h)
 AM_INIT_AUTOMAKE(xmlrpc, 0.54)
 
-AM_PROG_LIBTOOL
-
 dnl Checks for programs.
 AC_PROG_AWK
 AC_PROG_CC
@@ -52,7 +50,7 @@
 AC_FUNC_STRFTIME
 
 AC_CHECK_FUNCS(mktime strstr)
+AC_CHECK_FUNC(iconv,[],[AC_CHECK_LIB(iconv,libiconv,[],[AC_MSG_ERROR([Sorry libiconv not found, please ensure libiconv/libiconv-dev is installed])])])
 
-AC_OUTPUT()
 AC_OUTPUT(src/Makefile Makefile sample/Makefile)
 
--- origsrc/xmlrpc-epi-0.54/sample/Makefile.am	2008-04-14 07:05:49.000000000 -0500
+++ src/xmlrpc-epi-0.54/sample/Makefile.am	2008-05-07 16:35:43.734375000 -0500
@@ -1,7 +1,7 @@
 
-INCLUDES=-I../src $(EXTRA_INCLUDES) 
+INCLUDES=-I$(top_srcdir)/src $(EXTRA_INCLUDES) 
 
-bin_PROGRAMS = sample client server server_compliance_test memtest hello_client hello_server
+noinst_PROGRAMS = sample client server server_compliance_test memtest hello_client hello_server
 
 sample_SOURCES = sample.c
 sample_LDADD = -L@prefix@/lib ../src/libxmlrpc-epi.la
--- origsrc/xmlrpc-epi-0.54/src/Makefile.am	2008-04-16 05:38:07.000000000 -0500
+++ src/xmlrpc-epi-0.54/src/Makefile.am	2008-05-07 19:07:45.453125000 -0500
@@ -16,9 +16,9 @@
 	xmlrpc_introspection.c \
         system_methods.c
         
-libxmlrpc_epi_la_LDFLAGS = -version-info 0:3:0
-libxmlrpc_epi_incdir=$(includedir)/xmlrpc-epi 
-include_HEADERS = \
+libxmlrpc_epi_la_LDFLAGS = -version-info 0:3:0 -no-undefined
+libxmlrpc_epi_incdir=$(includedir)/xmlrpc-epi
+libxmlrpc_epi_inc_HEADERS = \
 	base64.h \
 	encodings.h \
 	queue.h \
